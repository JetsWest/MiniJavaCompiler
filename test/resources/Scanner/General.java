int a = 17;
int b = 10 * a;
int c = a + b;
System.out.println(c);