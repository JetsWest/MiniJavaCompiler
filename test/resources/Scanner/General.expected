INT ID(a) BECOMES DIGIT(17) SEMICOLON INT ID(b) BECOMES DIGIT(10) MULT ID(a) SEMICOLON INT ID(c) BECOMES ID(a) PLUS ID(b) SEMICOLON PRINT LPAREN ID(c) RPAREN SEMICOLON